pkgs/servers/matrix-synapse/default.nix

```diff
@@ -1,27 +1,13 @@
-{ lib, stdenv, python3, openssl
-, enableSystemd ? stdenv.isLinux
+{ stdenv, lib, python, buildPythonApplication, fetchPypi
```

Comment:
`python` seems to be an alias for `python2`, not `python3`.
https://github.com/NixOS/nixpkgs/blob/daac44f/pkgs/top-level/all-packages.nix#L8873

Reply:
Nevermind, just noticed that the package was called with `python3.pkgs.callPackage` in `top-level/all-packages.nix`.